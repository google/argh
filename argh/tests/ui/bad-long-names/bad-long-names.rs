/// Command
#[derive(argh::FromArgs)]
struct Cmd {
    #[argh(switch)]
    /// non-ascii
    привет: bool,
    #[argh(switch)]
    /// uppercase
    XMLHTTPRequest: bool,
    #[argh(switch, long = "not really")]
    /// bad attr
    ok: bool,
}

fn main() {}
