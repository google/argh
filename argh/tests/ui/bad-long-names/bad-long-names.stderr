error: Long names must be ASCII
 --> tests/ui/bad-long-names/bad-long-names.rs:6:5
  |
6 |     привет: bool,
  |     ^^^^^^

error: Long names must be lowercase
 --> tests/ui/bad-long-names/bad-long-names.rs:9:5
  |
9 |     XMLHTTPRequest: bool,
  |     ^^^^^^^^^^^^^^

error: Long names must be lowercase
  --> tests/ui/bad-long-names/bad-long-names.rs:10:27
   |
10 |     #[argh(switch, long = "not really")]
   |                           ^^^^^^^^^^^^
